src/apps/content-editor/src/app/views/ItemEdit/Content/Actions/Widgets/WidgetDraftHistory/WidgetDraftHistory.js

```diff
@@ -53,21 +48,35 @@ export default connect((state, props) => {
         {loading ? (
           <p>Loading Logs</p>
         ) : props.logs.length ? (
-          <ul className="logs">
-            {props.logs.map((log) => (
-              <li className="log" key={log.ZUID}>
-                <strong>
-                  {log.firstName} {log.lastName}
-                </strong>
-                &nbsp;
-                <span>saved {moment(log.happenedAt).fromNow()}</span>
-              </li>
-            ))}
-          </ul>
+          <>
+            <ul className="logs">
+              {props.logs.map((log) => (
+                <li className="log" key={log.ZUID}>
+                  <strong>
+                    {log.firstName} {log.lastName}
+                  </strong>
+                  &nbsp;
+                  <span>modified {moment(log.happenedAt).fromNow()}</span>
+                </li>
+              ))}
+            </ul>
+            {/* TODO: Replace this link with Zesty MUI AppLink */}
+            <Link
+              color="#FF5D03"
+              underline="hover"
+              href="#"
+              onClick={(evt) => {
+                evt.preventDefault();
+                history.push(
+                  `/reports/activity-log/resources/${props.itemZUID}`
```

Comment:
I won't mark this as blocking to deploy but we need a better abstraction for internal app link routing than supplanting default a tag behavior.

Reply:
Agreed I believe its marked with a TODO to replace with the yet to be completed Zesty MUI AppLink